app/controllers/api/verify/password_confirm_controller.rb

```diff
@@ -59,6 +60,52 @@ def in_person_enrollment?(user)
         # WILLFIX: After LG-6872 and we have enrollment saved, reference enrollment instead.
```

Comment:
Anything we need to think about here? I think the assumption at the time this was written was that we might create an enrollment earlier in the process.

Reply:
I think it's fine as-is. This method gets used twice: once before the enrollment is created and once after. The enrollment is only a useful indicator of this being an in-person flow after it is created

I'm going to leave it as-is in this file and will begin using the enrollment on the next page, in the person key controller